src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleInput.cs

```diff
@@ -28,7 +28,7 @@ public void CustomizeView(DoubleInput nodeModel, NodeView nodeView)
 
             tb.DataContext = nodeModel;
 
-            tb.BindToProperty(new Binding("Value")
+            tb.BindToProperty(new Binding("NumericalValue")
             {
                 Mode = BindingMode.TwoWay,
                 Converter = new DoubleInputDisplay(),
```

Comment:
We are already using a converter for this value? Can you append your logic to that converter? If we don't do this, we now have input conversion happening in two places, which will make this hard to debug.